assets/xml/objects/object_tokei_turret.xml

```diff
@@ -1,17 +1,17 @@
 ﻿<Root>
     <File Name="object_tokei_turret" Segment="6">
         <Texture Name="object_tokei_turret_TLUT_000000" OutName="tlut_000000" Format="rgba16" Width="4" Height="4" Offset="0x0" />
-        <Texture Name="object_tokei_turret_Tex_000020" OutName="tex_000020" Format="ci4" Width="64" Height="64" Offset="0x20" />
-        <Texture Name="object_tokei_turret_Tex_000820" OutName="tex_000820" Format="rgba16" Width="32" Height="32" Offset="0x820" />
-        <Texture Name="object_tokei_turret_Tex_001020" OutName="tex_001020" Format="rgba16" Width="16" Height="32" Offset="0x1020" />
-        <Texture Name="object_tokei_turret_Tex_001420" OutName="tex_001420" Format="rgba16" Width="128" Height="16" Offset="0x1420" />
+        <Texture Name="gTokeiTurretTextTex" OutName="tokei_turret_text" Format="ci4" Width="64" Height="64" Offset="0x20" />
+        <Texture Name="gTokeiTurretFabricTex" OutName="tokei_turret_fabric" Format="rgba16" Width="32" Height="32" Offset="0x820" />
+        <Texture Name="gTokeiTurretWoodTex" OutName="tokei_turret_wood" Format="rgba16" Width="16" Height="32" Offset="0x1020" />
+        <Texture Name="gClockTownFlagsTex" OutName="clock_town_flags" Format="rgba16" Width="128" Height="16" Offset="0x1420" />
         <DList Name="object_tokei_turret_DL_002500" Offset="0x2500" />
```

Comment:
You might as well at least note that the 3 unnamed DLs are empty.

Reply:
Can just name them `gTokeiTurretEmpty#DL` where `#` is just the number order they appear in. For example see object_ctower_rot.xml